lldb/source/Plugins/Process/elf-core/ThreadElfCore.cpp

```diff
@@ -584,22 +584,26 @@ Status ELFLinuxSigInfo::Parse(const DataExtractor &data, const ArchSpec &arch,
   // 64b ELF have a 4 byte pad.
   if (data.GetAddressByteSize() == 8)
     offset += 4;
-  // Not every stop signal has a valid address, but that will get resolved in
-  // the unix_signals.GetSignalDescription() call below.
-  if (unix_signals.GetShouldStop(si_signo)) {
+
+  if (si_code < 0) {
```

Comment:
So this check is my primary concern with my own code. In the Linux source I linked they define user space as `<0`, and I followed suite here. I would ideally like to make this a static function on LinuxSignals, but wanted to get feedback on how we want to implement this.